server/routes/upload.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const router = express.Router();
+
+const upload = require('../services/upload-image');
+
+const uploadSingleImage = upload.single('image');
+
+router.route('/image').post((req, res) => {
+  uploadSingleImage(req, res, function(err) {
+    console.log(err);
+    if (err) {
+      return res.status(422).send({
+        errors: [{
+          title: 'file upload error',
+          detail: err.message
+        }]
+      })
+    }
+    return res.json({'imageUrl': req.file.location});
+  });
```

Comment:
Can you move this logic into a controller file to match the conventions from other routes?